### PR TITLE
added genshin api explainaiton for most endpoints(will add more next few days)

### PR DESCRIPTION
**Pull request by teehee567**
will add hsr next few days

### CODE REVIEW

**Comment by ashleney** [comment]
Could you make this part of the actual docs so that it can also be on GitHub pages

**Comment by teehee567** [comment]
sure, il move it

File: .DS_Store
Comment (FichteFoll) [inline .DS_Store:1]: I suggest to add `.DS_Store` to your [global .gitignore](https://stackoverflow.com/a/7335487).

Comment (teehee567) [reply .DS_Store:1]: will do, i usually dont work on my mac lol

**Comment by ashleney** [comment]
Could you maybe make some of these collapsible with <details>

**Comment by ashleney** [comment]
Can we consider this PR abandoned?

**Pull request closed without merging**